components/accounts/AddressParser.vue

```diff
@@ -29,3 +29,9 @@ export default class AddressParser extends Vue {
   }
 }
 </script>
+
+<style lang="scss">
+  .ap-textarea {
+    height: 400px !important;
+  }
+</style>
```

Comment:
Why did you need to specify height of textare?

Reply:
@roiLeo because the height of current textarea is small and @yangwao mentioned to make the send window at least three times height of original one.